app/src/operations/mail/index.ts

```diff
@@ -14,45 +14,76 @@ export default defineOperationApp({
 			label: '$t:operations.mail.to',
 			text: Array.isArray(to) ? to.join(', ') : to,
 		},
+		{
+			label: '$t:type',
+			text: type,
+		},
 		{
 			label: '$t:operations.mail.body',
 			text: body,
 		},
 	],
-	options: [
-		{
-			field: 'to',
-			name: '$t:operations.mail.to',
-			type: 'csv',
-			meta: {
-				width: 'full',
-				interface: 'tags',
-				options: {
-					placeholder: '$t:operations.mail.to_placeholder',
-					iconRight: 'alternate_email',
+	options: (panel) => {
+		return [
+			{
+				field: 'to',
+				name: '$t:operations.mail.to',
+				type: 'csv',
+				meta: {
+					width: 'full',
+					interface: 'tags',
+					options: {
+						placeholder: '$t:operations.mail.to_placeholder',
+						iconRight: 'alternate_email',
+					},
 				},
 			},
-		},
-		{
-			field: 'subject',
-			name: '$t:subject',
-			type: 'string',
-			meta: {
-				width: 'full',
-				interface: 'input',
-				options: {
-					iconRight: 'title',
+			{
+				field: 'subject',
+				name: '$t:subject',
+				type: 'string',
+				meta: {
+					width: 'full',
+					interface: 'input',
+					options: {
+						iconRight: 'title',
+					},
 				},
 			},
-		},
-		{
-			field: 'body',
-			name: '$t:operations.mail.body',
-			type: 'text',
-			meta: {
-				width: 'full',
-				interface: 'input-rich-text-md',
+			{
+				field: 'type',
+				name: '$t:type',
+				type: 'string',
+				schema: {
+					default_value: 'input-rich-text-md',
+				},
+				meta: {
+					interface: 'select-dropdown',
+					width: 'full',
+					required: true,
+					options: {
+						choices: [
+							{
+								text: '$t:interfaces.input-rich-text-md.markdown',
+								value: 'input-rich-text-md',
+							},
+							{
+								text: '$t:interfaces.input-rich-text-html.wysiwyg',
+								value: 'input-rich-text-html',
```

Comment:
@rijkvanzanten thoughts on using values `markdown`/`wysiwyg` instead of `input-rich-text-md`/`input-rich-text-html`? So that in the API email operation we'd do `type === 'wysiwyg'` instead of `type === 'input-rich-text-html'`. 

Technically this suggestion might be redundant, but more so thinking about the raw value being more "meaningful"/readable for users that may not correlate `input-rich-text-html` as WYSIWYG. (also might be abit more relevant if we opt to show `type` in the operation overview)

Reply:
Alright, gotcha